Documentation-Requirements.md

```diff
@@ -0,0 +1,60 @@
+* [Introduction](#introduction)
+* [General requirements](#general-requirements)
+* [Code blocks](#code-blocks)
+
+# Introduction
+
+This document outlines the requirements for all documentation in the [Kata
+Containers](https://github.com/kata-containers) project.
+
+# General requirements
+
+- All documents are expected to be written in GitHub Flavored Markdown format.
```

Comment:
You could hotlink to [Github Flavored Markdown](https://github.github.com/gfm/)

Reply:
Good point - done.